ext/liquid_c/variable_lookup.c

```diff
@@ -60,5 +60,6 @@ void init_liquid_variable_lookup()
     id_ivar_command_flags = rb_intern("@command_flags");
 
     VALUE cLiquidVariableLookup = rb_const_get(mLiquid, rb_intern("VariableLookup"));
+    rb_global_variable(&cLiquidVariableLookup);
```

Comment:
Did we really have to mark this object? It looks like it is just a local variable access which is not passing pointers around. Am I missing something here?

Reply:
No, we only need this for global variables, so we shouldn't do it here.